metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_about.md

```diff
@@ -6,6 +6,8 @@ subclass Phyllopoda. It inhabits a variety of freshwater environments
 and is broadly distributed throughout the northern hemisphere, and in
 South Africa.
 
+D. magna is used in many different fields of research, such as ecotoxicology, population genetics, the evolution of sex, phenotypic plasticity, ecophysiology (including global change biology) and host-parasite interactions.
```

Comment:
Cheers Vasily. Just to add a bit more as its an important model organism :)

Reply:
updated with the "gca" one